.github/workflows/ci.yaml

```diff
@@ -29,3 +29,19 @@ jobs:
         [
           {"folder": ".", "bzlmodEnabled": false}
         ]
+  # For branch protection settings, this job provides a "stable" name that can be used to gate PR merges
+  # on "all matrix jobs were successful".
+  conclusion:
+    needs: test
+    runs-on: ubuntu-latest
+    if: always()
+    steps:
+      - uses: technote-space/workflow-conclusion-action@45ce8e0eb155657ab8ccf346ade734257fd196a5 # v3.0.3
+
+      - if: ${{ env.WORKFLOW_CONCLUSION == 'success' }}
+        working-directory: /tmp
+        run: echo ${{ env.WORKFLOW_CONCLUSION }} && exit 0
```

Comment:
Can we set a `name` on these steps? Something like `Handle failure` and `Handle success`? Currently for the failure case that does not execute in the [demo](https://github.com/bazel-contrib/rules-template/actions/runs/9783272801/job/27011588324?pr=124) you posted it evaluates `env.WORKFLOW_CONCLUSION` in the name which is confusing.

Reply:
good point